src/Network/Network/LoadBalancer/GetAzureLoadBalancerBackendPool.cs

```diff
@@ -0,0 +1,115 @@
+    {
+
+        private const string GetByNameParameterSet = "GetByNameParameterSet";
+        private const string GetByParentObjectParameterSet = "GetByParentObjectParameterSet";
+        private const string GetByResourceIdParameterSet = "GetByResourceIdParameterSet";
+
+
+        [Parameter(Mandatory = true, HelpMessage = "The resource group name of the load balancer.", ParameterSetName = GetByNameParameterSet)]
+        [ResourceGroupCompleter]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(Mandatory = true, HelpMessage = "The name of the load balancer.", ParameterSetName = GetByNameParameterSet)]
+        [ValidateNotNullOrEmpty]
+        public string LoadBalancerName { get; set; }
+
+
+        [Parameter(Mandatory = false, HelpMessage = "The name of the backend address pool.", ParameterSetName = GetByNameParameterSet)]
+        [Parameter(Mandatory = false, HelpMessage = "The name of the backend address pool.", ParameterSetName = GetByParentObjectParameterSet)]
+        [ValidateNotNullOrEmpty]
+        public string Name {get; set; }
+
+        [Parameter(Mandatory = true, ValueFromPipeline = true, ParameterSetName = GetByParentObjectParameterSet)]
+        [ValidateNotNull]
+        public PSLoadBalancer LoadBalancer { get; set; }
+
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, ParameterSetName = GetByResourceIdParameterSet)]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+
+        public override void Execute()
+        {
+            base.Execute();
+            BackendAddressPool loadBalancerBackendAddressPool = null;
+
+
+            if (this.IsParameterBound(c => c.LoadBalancer))
+            {
+                this.ResourceGroupName = this.LoadBalancer.ResourceGroupName;
+                this.LoadBalancerName = this.LoadBalancer.Name;
+            }
+
+            if (this.IsParameterBound(p => p.ResourceId))
+            {
+                ResourceIdentifier resourceIdentifier = new ResourceIdentifier(ResourceId);
+                this.ResourceGroupName = resourceIdentifier.ResourceGroupName;
+                this.LoadBalancerName = resourceIdentifier.ParentResource.Split('/')[1];
+                this.Name = resourceIdentifier.ResourceName;
+            }
+
+
+            // Get single backend pool
+            if (ShouldGetByName(this.ResourceGroupName, this.Name))
+            {
+                loadBalancerBackendAddressPool = this.NetworkClient.NetworkManagementClient.LoadBalancerBackendAddressPools.Get(this.ResourceGroupName, this.LoadBalancerName, this.Name);
+
+                var loadBalancerBackendAddressPoolModel = NetworkResourceManagerProfile.Mapper.Map<PSBackendAddressPool>(loadBalancerBackendAddressPool);
+                loadBalancerBackendAddressPoolModel.LoadBalancerBackendAddresses = loadBalancerBackendAddressPoolModel.LoadBalancerBackendAddresses.ToList();
```

Comment:
why do we need to set the load balancer backend addresses?

Reply:
I removed this.